curriculum/challenges/english/08-coding-interview-prep/rosetta-code/run-length-encoding.md

```diff
@@ -0,0 +1,128 @@
+---
+id: 5eb257ba998e23057d209754
+title: Run-length Encoding
+challengeType: 5
+---
```

Comment:
I just create a [guide article](https://www.freecodecamp.org/forum/t/freecodecamp-challenge-guide-run-length-encoding/387031) on the forum for this challenge. Could you add `forumTopicId` to the frontmatter?:

````
---
id: 5eb257ba998e23057d209754
title: Run-length Encoding
challengeType: 5
forumTopicId: 387031
---
````